_includes/features.html

```diff
@@ -1,5 +1,5 @@
 <section id="features">
-  <h1 class="claim-title">We believe a <span class="highlight">trustless</span>, <span class="highlight">private</span> economy can be built <span class="highlight">without middlemen</span></h1>
+  <h1 class="claim-title"> <span class="highlight">One</span> breakthrough platform, <span class="highlight">endless</span> possibilities</h1>
```

Comment:
I personally prefer the old one, which is more concrete. We should stick to more specific texts instead of these abstract ones IMO.

Reply:
> We believe a more efficient economy can be built without middlemen

That sound nice :)